### PR TITLE
Feature version check

### PR DESCRIPTION
**Pull request by domtancredi**
This is really close and should be tested. It checks for version (in a new version endpoint), refactors toast into its own script, and has a small component for comparing version and displaying a toast if they don't match.

There's some issue with how to check app-state with the version number in the jade file so that's something to look into.

### CODE REVIEW

**Comment by domtancredi** [comment]
Based on the feature branch made before a bunch of updates to master, it may make more sense to drop and re-build the feature branch from a new place in master

**Comment by zaroth** [comment]
yeah, I would hate to lose the sweet upgrades made to toasts that @saintis coded today.

**Comment by Saintis** [comment]
How easy would it be to bring this PR up to speed with `master`?

**Comment by domtancredi** [comment]
Not too bad. The changes were minor, on the template level as well as the server-side area (didn't touch core game engine).

**Comment by domtancredi** [comment]
Done. Could use a review @Saintis

**Comment by Saintis** [comment]
It doesn't look as though the toast function correctly allows the toaster-options to be passed in. 

Also, does the version number have to be manually updated or can it be done automatically somehow?

File: src/cljs/netrunner/main.cljs [hunk 1]
```diff
@@ -28,6 +31,15 @@
 (.setPathPrefix history "")
 (.setEnabled history true)
 
+(go (while true
```
Comment (nealterrell) [inline src/cljs/netrunner/main.cljs]: This loop is currently in `gameboard.cljs` -- why is it here too?

**Comment by nealterrell** [comment]
RE: version number, he is loading it server-side from `app.locals.version`, which is read from the environment variable `APP_VERSION`. @mtgred updates this by hand when deploying a UI update.

How exactly does this work, @domtancredi? When does the warning about bad version show up?

**Comment by nealterrell** [comment]
We can revisit this with the new Clojure server anytime, but the actual code here is dead I think.

**Pull request closed without merging**